Cargo.toml

```diff
@@ -29,12 +29,15 @@ libtor = { version = "47.13.0", optional = true, features = ["vendored-openssl"]
 mitosis = { version = "0.1.1", optional = true }
 log4rs = "1.3.0"
 openssl-sys = { version = "0.9.68", optional = true }
+flate2 = {version = "1.0.35", optional = true}
+tar = {version = "0.4.43", optional = true}
+minreq = { version = "2.12.0", features = ["https"] , optional = true}
```

Comment:
Can we add these as dev-dependcies? we don't need them in production. Even better, can we avoid them? 

like flate2 and tar are for compression, what if we don't compress at source at all? 

minreq for HTTP, but we are using our custom get/post message at DNS, can we do the same here?

Reply:
Ack.